pq-crypto/kyber_r3/kyber512r3_reduce.c

```diff
@@ -1,6 +1,8 @@
+#include "kyber512r3_reduce.h"
+
```

Comment:
I'm guessing clang-format got at this file? Is that necessary? Isn't this copy/pasted from the reference implementation? It doesn't have a copyright